<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Азбука Морзе</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/img/logo.png</normaloff>:/img/logo.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>34</width>
    <height>34</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QComboBox" name="comboBox">
            <item>
             <property name="text">
              <string>Текст → код Морзе</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Код Морзе → английский язык</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Код Морзе → русский язык</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="textInputFields">
          <property name="minimumSize">
           <size>
            <width>256</width>
            <height>180</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>MS Shell Dlg 2</family>
            <pointsize>12</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>22</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>MS Sans Serif</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Азбука Морзе</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>22</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextBrowser" name="textBrowser">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="font">
           <font>
            <family>MS Shell Dlg 2</family>
            <pointsize>12</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="sound"/>
   <addaction name="separator"/>
   <addaction name="nightMode"/>
   <addaction name="separator"/>
   <addaction name="history"/>
   <addaction name="separator"/>
   <addaction name="actionSaveInHistory"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="openFile"/>
    <addaction name="saveFile"/>
    <addaction name="exit"/>
   </widget>
   <widget class="QMenu" name="view">
    <property name="title">
     <string>Вид</string>
    </property>
    <widget class="QMenu" name="language">
     <property name="title">
      <string>Выбор языка</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>img/icons8-language-100.png</normaloff>img/icons8-language-100.png</iconset>
     </property>
     <addaction name="toRussion"/>
     <addaction name="toEnglish"/>
    </widget>
    <widget class="QMenu" name="Thems">
     <property name="title">
      <string>Выбор темы</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>img/wive_icon.svg</normaloff>img/wive_icon.svg</iconset>
     </property>
     <addaction name="Fusion"/>
     <addaction name="Windows"/>
     <addaction name="WindowsXP"/>
     <addaction name="WindowVista"/>
    </widget>
    <addaction name="language"/>
    <addaction name="Thems"/>
    <addaction name="night_mode"/>
   </widget>
   <widget class="QMenu" name="control">
    <property name="title">
     <string>Управление</string>
    </property>
    <addaction name="sound"/>
    <addaction name="history_2"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>Изучения/тест</string>
    </property>
    <widget class="QMenu" name="menu_6">
     <property name="title">
      <string>Тест</string>
     </property>
     <addaction name="letters"/>
    </widget>
    <addaction name="study"/>
    <addaction name="menu_6"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="control"/>
   <addaction name="view"/>
   <addaction name="menu_5"/>
  </widget>
  <action name="exit">
   <property name="icon">
    <iconset>
     <normaloff>img/Icon_close.svg</normaloff>img/Icon_close.svg</iconset>
   </property>
   <property name="text">
    <string>Выйти</string>
   </property>
   <property name="toolTip">
    <string>Выход из приложения</string>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="sound">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/sound.png</normaloff>:/img/sound.png</iconset>
   </property>
   <property name="text">
    <string>Воспроизвести</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="toRussion">
   <property name="icon">
    <iconset>
     <normaloff>img/russia-flag-icon-128.png</normaloff>img/russia-flag-icon-128.png</iconset>
   </property>
   <property name="text">
    <string>Русский язык</string>
   </property>
  </action>
  <action name="toEnglish">
   <property name="icon">
    <iconset>
     <normaloff>img/united-states-of-america-flag-icon-128.png</normaloff>img/united-states-of-america-flag-icon-128.png</iconset>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="toTojik">
   <property name="icon">
    <iconset>
     <normaloff>img/tajikistan-flag-icon-128.png</normaloff>img/tajikistan-flag-icon-128.png</iconset>
   </property>
   <property name="text">
    <string>Забони тоҷикӣ</string>
   </property>
  </action>
  <action name="openFile">
   <property name="icon">
    <iconset>
     <normaloff>img/Tilda_Icons_30_system_folder.svg</normaloff>img/Tilda_Icons_30_system_folder.svg</iconset>
   </property>
   <property name="text">
    <string>Открыть из файла</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="saveFile">
   <property name="icon">
    <iconset>
     <normaloff>img/Tilda_Icons_30_system_save.svg</normaloff>img/Tilda_Icons_30_system_save.svg</iconset>
   </property>
   <property name="text">
    <string>Сохранить в файл</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="Fusion">
   <property name="text">
    <string>Fusion</string>
   </property>
  </action>
  <action name="Windows">
   <property name="text">
    <string>Windows</string>
   </property>
  </action>
  <action name="WindowsXP">
   <property name="text">
    <string>Windows Xp</string>
   </property>
  </action>
  <action name="WindowVista">
   <property name="text">
    <string>Window Vista</string>
   </property>
  </action>
  <action name="letters">
   <property name="text">
    <string>Буквы</string>
   </property>
  </action>
  <action name="words">
   <property name="text">
    <string>Слова</string>
   </property>
  </action>
  <action name="sentences">
   <property name="text">
    <string>Предложения</string>
   </property>
  </action>
  <action name="nightMode">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/dark-mode.png</normaloff>:/img/dark-mode.png</iconset>
   </property>
   <property name="text">
    <string>Dark</string>
   </property>
  </action>
  <action name="night_mode">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/dark-mode.png</normaloff>:/img/dark-mode.png</iconset>
   </property>
   <property name="text">
    <string>Ночной режим</string>
   </property>
  </action>
  <action name="study">
   <property name="text">
    <string>Изучит</string>
   </property>
  </action>
  <action name="history">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/history.png</normaloff>:/img/history.png</iconset>
   </property>
   <property name="text">
    <string>История</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="history_2">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/history.png</normaloff>:/img/history.png</iconset>
   </property>
   <property name="text">
    <string>История</string>
   </property>
  </action>
  <action name="actionSaveInHistory">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/save.png</normaloff>:/img/save.png</iconset>
   </property>
   <property name="text">
    <string>SaveInHistory</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>textInputFields</tabstop>
  <tabstop>textBrowser</tabstop>
 </tabstops>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
